### ISSUE TITLE
Do not skip html heading levels

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1665]
Relevant Components
- [ ] Document Information
- [ ] File Upload

Description
Missing a heading level (such as having an ```<h4>``` follow an ```<h1>```) isn’t a WCAG failure, but it isn’t good practice. 

Current Code
```<h4>```

Instructions:
```</h4>```
Solution
This issue could be resolved by changing the ```<h4>``` in this example to an ```<h2>```.  However, you might also look at the page’s current structure and find other areas to improve keyboard navigation.

**Comment by jcreel** [comment]
Deliverable:  investigate HTML to see where mis-ordering of h elements is happening.  Provide a decomposition and sizing of the task to remediate.
